src/main.cpp

```diff
@@ -8,8 +12,10 @@ int main(int argc, char** argv)
 		return 1;
 	}
 	(void)argv;
```

Comment:
Also here the same as before, using c++ static void casting

Reply:
Well, when we implement the config file we won't have to cast anyway :D
still: changed